### PR TITLE
Paging via Strategy

### PR DESCRIPTION
**Pull request by haubitzer**

### CODE REVIEW

File: src/initialLoad/initialLoadEpic.ts [hunk 1]
```diff
@@ -13,21 +13,23 @@ import { usersSlice } from '../entities/users/usersSlice';
 import { RootState } from '../rootState';
 import { initialLoadSlice } from './initialLoadSlice';
 
+const INITIAL_PAGE_SIZE = 100;
```
Comment (haubitzer) [inline src/initialLoad/initialLoadEpic.ts:16]: Page size should be changed later to 2000 but at the moment it helps for tests/debugging

File: src/initialLoad/initialLoadEpic.ts [hunk 2]
```diff
@@ -13,21 +13,23 @@ import { usersSlice } from '../entities/users/usersSlice';
...
   action$.pipe(
     filter(initialLoadSlice.actions.start.match),
     mergeMap((_action) =>
       from([
-        absenceTypesSlice.actions.requestInit(),
-        generalSettingsSlice.actions.requestInit(),
-        projectsSlice.actions.requestInit(),
-        tasksSlice.actions.requestInit(new RequestParams<Task>().limit(2000)),
-        usersSlice.actions.requestInit(),
+        absenceTypesSlice.actions.initialLoad(new RequestParams<AbsenceType>().limit(INITIAL_PAGE_SIZE)),
```
Comment (benj56) [inline src/initialLoad/initialLoadEpic.ts:23]: None of those will work in the desktop app due to the non-serializable actions problem

**Review by benj56 — CHANGES_REQUESTED** [review]
LGTM, fixed some types to get rid of the eslint disables/any. I committed the type fixes in that branch: implement-paging-via-epicStrategyBuilder-with-type-fixes

I think there are 2 problems we need to investigate next week before merging:

- placement of the catchError, I believe it breaks after the first error
- unserializable Actions

**Comment by haubitzer** [comment]
@benj56 Thank you for your feedback. I like your changes. Please feel free to apply them and merge this story.

**Review by benj56 — APPROVED** [review]

**Review by sergej-tschak — APPROVED** [review]

**Pull request merged**